src/main/java/com/faforever/api/league/domain/LeagueSeasonDivisionSubdivision.java

```diff
@@ -0,0 +1,73 @@
+package com.faforever.api.league.domain;
+
+import com.yahoo.elide.annotation.Include;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Setter
+@Table(name = "league_season_division_subdivision")
+@Include(type = LeagueSeasonDivisionSubdivision.TYPE_NAME)
+public class LeagueSeasonDivisionSubdivision {
+  public static final String TYPE_NAME = "leagueSeasonDivisionSubdivisionV2";
```

Comment:
Any reason for v2? We never launched a V1

Reply:
renamed